src/components/Graph.js

```diff
@@ -8,20 +8,7 @@ import { DateTimePicker } from './DateTimePicker';
 import { NodeDetailCard } from './NodeDetailCard';
 import { IconMap } from './IconMap';
 import './Graph.scss';
-
-function truncate(fullStr, strLen, separator='...') {
-  if (fullStr.length <= strLen) return fullStr;
-
-  const sepLen = separator.length,
-    charsToShow = strLen - sepLen,
-    frontChars = Math.ceil(charsToShow / 2),
-    backChars = Math.floor(charsToShow / 2);
-
-  return fullStr.substr(0, frontChars) +
-    separator +
-    fullStr.substr(fullStr.length - backChars);
-}
-
+import { truncate, clearClicked, scaleGraph, drag, prepareGraphData } from './GraphUtils.js';
```

Comment:
This one is rather a question than a suggestion. I am not very familiar with React. Is it a good practice to import individual methods vs. the entire module and call methods using `utils.truncate`, `utils.scaleGraph`, etc.?

Reply:
Yes, the both method are used and it depends on number of methods and project assumptions. After review refactor I imported entire module in case of using multiple functions from it.